ui/src/models/user.js

```diff
@@ -28,3 +28,61 @@ export function signIn(token) {
         return response;
     });
 }
+
+export function deleteQuestion(questionId) {
+    const options = {
+        url: `/api/users/me/questions/${questionId}`,
+        method: 'DELETE'
+    };
+
+    return ApiFetch(options).then((response) => {
+        if (response.error) {
+            return;
+        }
+        return response;
+    });
+}
+
+export function updateQuestion(questionId, body) {
+    const options = {
+        url: `/api/users/me/questions/${questionId}`,
+        method: 'PUT',
+        body: body
+    };
+
+    return ApiFetch(options).then((response) => {
```

Comment:
quite repetitive, maybe find a way to reuse the default if statements if(error) then ..

Reply:
Fixed